CMakeLists.txt

```diff
@@ -363,6 +363,14 @@ include_directories("${CMAKE_SOURCE_DIR}")
 include_directories("/usr/local/include")
 link_directories("/usr/local/lib")
 
+# Boost 1.60.0 fails on variadic templates for Apple Clang
+if(APPLE)
```

Comment:
The gate for Apple is because the boost package may not have installed a CMake find package file?